### ISSUE TITLE
Links text and buttons UI are Overlapping with Zoom 200%, 400%, Text Scaling & High Dpi

### ISSUE THREAD

**Issue by msftedad** [docsifyjs/docsify#1853]
### Environment Details ###
OS version: Windows11 version 21H2 (OS build 22000.856)
Edge version: 104.0.1293.47 

### Repro Steps: ### 

1. Open URL: [ Document (1).mhtml](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmicrosofteur-my.sharepoint.com%2F%3Au%3A%2Fg%2Fpersonal%2Fa-zjankovic_microsoft_com%2FEXqO1n1Hk8dMqJu7z0IEt0QBDZc920MJJKp6pgRnMgsQ1w&data=05%7C01%7Cv-jjayaram%40microsoft.com%7Cbb5cb060322147c10c2208da78fe2f41%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955330465685284%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=WLklQ65FGfusI3YLPWXYmPS1dDzr02c1GbpVBEWP4HQ%3D&reserved=0)
2. Apply text size to 200% and resolution 1280x720. 
3. And observe that "my heading is a list item" and "example" texts are getting to overlap

### Actual Result: ###
After resizing and reflowing text is getting to overlap

### Expected Result: ###
After resizing and reflowing text should not get overlapped

### Screenshot Attachment: ###
<img width="851" alt="image" src="https://user-images.githubusercontent.com/93735775/184037208-ef8e3e74-86e6-4145-b251-1917e33cb32a.png">

**Comment by trusktr** [comment]
Hello! The link is not working. It takes me to a Microsoft login page.

**Comment by msftedad** [comment]
Hi Team,

Here is the open URL to reproduce the issue https://zorana-jankovic.github.io/example/#/

**Comment by trusktr** [comment]
What's should the solution be?

**Comment by msftedad** [comment]
Hi,

This is the common way low vision users can see the content without confusion.

According to the https://www.w3.org/WAI/WCAG21/Understanding/resize-text and https://www.w3.org/WAI/WCAG21/Understanding/reflow WCAG rules the content should not get overlapped.

**Comment by v-sudmandal** [comment]
Hi @msftedad

Please let us know your plans on fixing this bug so that our team will regress and close it which makes the product Grade C

**Comment by SritejaGolla** [comment]
Hi @trusktr

Any update on the above bug ? Please let us know your plans on fixing this bug.

**Comment by v-sudmandal** [comment]
Hi @trusktr

Any update on the above bug?

**Comment by v-shamohsin** [comment]
Hello @trusktr 

Can you provide an update on this please

**Comment by v-shamohsin** [comment]
Hi @trusktr , Can you please provide an update on this please. Thanks

**Comment by v-shamohsin** [comment]
Hi @trusktr , Can you please provide an ETA for the fix

**Comment by msftedad** [comment]
Hi Team, is there any update on this bug?

**Comment by msftedad** [comment]
Hi Team, Is there any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Is there any update on this issue?

**Comment by jhildenbiddle** [comment]
Hello @msftedad --

This issue will not be completely resolved by PR #2469 but it will (hopefully) be a little easier to deal with. The changes will be available when Docsify v5 is released.

Once v5 is release, the navbar height can be adjusted easily using the following CSS custom property:

```css
:root {
  --navbar-height: 4em;
}
```

That said, stuffing a large number of elements into a horizontal navbar is going to be problematic regardless of the `--navbar-height` setting. I believe the intent of the original design (which was before my time) was to have either a limited number of root-level navbar items or to leverage drop-down menus in the when needed (as was done with the "Translations" menu on https://docsify.js.org):

<img src="https://github.com/user-attachments/assets/c38bb7a6-9577-4d6a-be75-17e9e34397bf">

```md
- Translations

  - [:uk: English](/)
  - [:cn: 简体中文](/zh-cn/)
  - [:de: Deutsch](/de-de/)
  - [:es: Español](/es/)
  - [:ru: Русский](/ru-ru/)
```

There are several ways we could resolve the issue you've described, but each one has trade-offs. We may consider some of these options in a future redesign, but for now it is unlikely that we will change the navbar style/behavior to "fix" the issue as described here because there are alternative approaches that avoid the issue.

**Comment by msftedad** [comment]
Thanks for the update.
